tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
+)
+
+func BenchmarkStress25Environments0Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(25, 0, 0, 0, deployCommand, b)
```

Comment:
Since `deployCommand` doesn't change, I think it makes sense to just hardcode it into the terraform configuration.

Reply:
✅

Reply:
Update: Since we should recreate zpatrick's old Task stress test with this newer sort of pattern, I'm going to declare two `const`s for deploy commands: a while loop to simulate a long-running service, and a simple sleep to simulate a one-off task. There will be a `DeployCommand` field in the `StressTestCase` struct which will reference one of these two `const`s.